### DISCUSSION TITLE
Real-time Generative Feedback Loop Automation with Supabase

### DISCUSSION THREAD

**Discussion by boburumurzokov** [supabase/supabase#28876] [Show and tell]
This blog post explores how to use  [Supabase](https://supabase.com/), [GlassFlow](https://www.glassflow.dev/), and [Weaviate](https://weaviate.io/) to process sample [Airbnb](https://www.airbnb.com/) listings in real-time, vectorizes long descriptions for optimized searching and personalized recommendations. The content can be specifically useful for existing Supabase and/or Weaviate users looking for solutions for detecting changes on the primary database, streaming these changes, and continuously updating the vector database for AI-powered applications.

![image](https://github.com/user-attachments/assets/e27ec390-d559-4538-8814-daaccab14b04)

Let me know if you have any feedback or improvement suggestions. Thanks!

**Comment by Boburmirzo** [comment]
If you're eager to get hands-on and create the pipeline yourself, you can follow the detailed steps in the documentation: https://docs.glassflow.dev/tutorials/use-cases/real-time-generative-feedback-loop-automation
